go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -84,27 +84,40 @@ func init() {
 	servenv.OnParseFor("vttablet", registerIncrementalRestoreFlags)
 
 	statsRestoreBackupTime = stats.NewString("RestoredBackupTime")
-	statsRestoreBackupPosition = stats.NewString("RestorePosition")
+	statsRestorePosition = stats.NewString("RestorePosition")
 }
 
-// RestoreData is the main entry point for backup restore.
-// It will either work, fail gracefully, or return
-// an error in case of a non-recoverable error.
+// RestoreBackup is the main entry point for backup restore. It will either
+// work, fail gracefully, or return an error in case of a non-recoverable error.
 // It takes the action lock so no RPC interferes.
-func (tm *TabletManager) RestoreData(
+func (tm *TabletManager) RestoreBackup(
```

Comment:
rename not strictly required, but i think it will help clarity once we add `RestoreClone` in next PR